plugins/gatsby-source-cd-airtable/tests/load-data.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { loadData } from '../src/load-data'
+
+jest.mock('node-fetch')
+
+import fetch from 'node-fetch'
+
+describe('loadData', () => {
+  test('should throw an error when no key is provided', () => {
+    process.env = {}
+    expect(() => loadData('tableName')).toThrow()
+  })
+
+  test('should throw and error when api key is not provided', () => {
+    process.env = {
+      AIRTABLE_BASE_KEY: 'key',
+    }
+    expect(() => loadData('tableName')).toThrow()
+  })
+
+  test('should throw and error when base key is not provided', () => {
+    process.env = {
+      AIRTABLE_API_KEY: 'key',
+    }
+    expect(() => loadData('tableName')).toThrow()
+  })
+
+  describe('with both keys provided', () => {
+    beforeEach(() => {
+      process.env = {
+        AIRTABLE_API_KEY: 'foo',
+        AIRTABLE_BASE_KEY: 'bar',
+      }
+    })
+
+    describe('data loading', () => {
+      test('should throw error when no table name is provided', () => {
+        expect(() => loadData((null as unknown) as string)).toThrow()
+      })
+      test('should send request to table url', () => {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        fetch.mockResolvedValue()
```

Comment:
Navrhoval bych vyhodit všechny do příslušných `before` bloků a zřetězit `mockResolvedValueOnce()`. Zároveň do `afterEach` přidat `jest.restoreAllMocks()`

Reply:
mohl bys @miiila prosím mrknout, jestli je to teď takhle ok?